src/documents/Mathematics.html.md.handlebars

```diff
@@ -103,3 +103,16 @@ Units and measurements poses their own unique challenge as units are often writt
 * For units within rest of content and outside of `<math>` blocks, use `<abbr>` for unit abbreviations with WAI-ARIA aria-label values. For example:
    * `<abbr role="text" title="kilograms" aria-label="kilograms">kg</abbr>`
    * `<abbr role="text" title="meters per second" aria-label="meters per second">m/s</abbr>`
+
+## MathML Cloud
+
+Benetech has developed a tool called <a href="https://www.mathmlcloud.org/" rel="nofollow" target="_blank" class="link-external">MathML Cloud</a> which allows content creators to convert math notation into readable text and other formats. MathML Cloud can take math notation in ASCII text, MathML or Latex notation, and output a text equivalent, PNG, SVG, and MML which can be embedded into EPUB 3 and HTML.
+
+<figure>
+	[![Screen capture of the MathML Cloud tool converting a math equation from ASCII into SVG, MML, and PNG formats.]({{{getRelativeUrl '/images/thumbs/400px-MathMLCloud.png' document.url}}})]({{{getRelativeUrl '/images/MathMLCloud.png' document.url}}})
+	<figcaption>
+		The MathML Cloud tool can convert math notation into readable text, SVG, MML, and PNG markup. The image shows the MathML interface converting an equation into different formats. Try MathMLCloud at <a href="https://www.mathmlcloud.org/" rel="nofollow" target="_blank" class="link-external">https://www.mathmlcloud.org/</a>
```

Comment:
The caption is a bit long. Is the first sentence necessary? And perhaps the link could be moved out of the caption?